main.js

```diff
@@ -1,7 +1,22 @@
 import { courses } from './courses.js'
 
+//Author: @Jacob Spires
+document
+  .getElementById('search-button')
+  .addEventListener('click', getCoursesFromTime)
+//getCoursesFromTime lets the user search for a course based on start time
+function getCoursesFromTime() {
+  let input = document.getElementById('timeSearch').value
+  input = input.toUpperCase() //accounts for lower case input
+  const rows = courses.map(course => {
+    //if input is in START_TIME, return the start time, course name, and course description
+    if (course.START_TIME.includes(input)) {
+      return `<tr><td>${course.START_TIME} - ${course.CRSE} - ${course.DESCR}</td></tr>`
+    }
+  })
+  document.querySelector('tbody').innerHTML = rows.join('')
+}
 const rows = courses.map(course => {
-  return `<tr><td>${course.CRSE} - ${course.DESCR}</td></tr>`
+  return `<tr><td>${course.START_TIME} - ${course.CRSE} - ${course.DESCR}</td></tr>`
```

Comment:
The reason I added the start time at the beginning was so that you could see all classes that began at that time and to test it for my own tests